File: JavaU2/P_7.java
//4- Concersor.

import java.util.Scanner;

public class Main
{
	public static void main(String[] args) {
		Scanner G = new Scanner(System.in);
		double gf;
		
		System.out.println("Ingresa los grados celcius a convertir: ");
		double gc = G.nextFloat();
		gf = (gc * 1.8) + 32;
		System.out.println(gc+"°C son iguales a: "+gf+"°F.");
	}
}

File: U2/P_7.cs
//5- Metros a pies.

using System;

class Metros {
  static void Main() {
      double f;
		
		Console.WriteLine("Ingresa los metros a convertir: ");
		double m = double.Parse(Console.ReadLine());
		f = m * 3.281;
		Console.WriteLine(m+"m son iguales a: "+f+"ft");
    }
}
